os/kernel/wdog/wd_dbg.c

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ *
+ * Copyright 2025 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <sys/types.h>
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdarg.h>
+#include <unistd.h>
+#include <sched.h>
+#include <assert.h>
+#include <errno.h>
+
+#include <tinyara/arch.h>
+#include <tinyara/wdog.h>
+#include <tinyara/mm/mm.h>
+
+#include "sched/sched.h"
+#include "wdog/wdog.h"
+
+#ifdef CONFIG_DEBUG
+
+static inline void wd_dump(struct wdog_s *wdog)
+{
+    int index = 0;
+	llwdbg("Wdog address = 0x%08x\n", wdog);
+	llwdbg("func: %p\n", wdog->func);
+	llwdbg("lag: %d\n", wdog->lag);
+	llwdbg("flags: %u\n", wdog->flags);
+	llwdbg("pid: %d\n", wdog->pid);
+	llwdbg("argc: %u\n", wdog->argc);
+    for (; index < CONFIG_MAX_WDOGPARMS; index++) {
+	    llwdbg("parm[%d]: %u\n", index, wdog->parm[index]);
+    }
+}
+
+void wd_corruption_dbg(struct wdog_s *wdog)
+{
+    uint32_t wd_node;
```

Comment:
Need to check indentation.
There are many lines which use spaces instead of tabs